homeassistant/components/broadlink/remote.py

```diff
@@ -83,6 +86,7 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
     """Set up the Broadlink remote."""
     host = config[CONF_HOST]
     mac_addr = config[CONF_MAC]
+    dev_type = config[CONF_TYPE]
```

Comment:
Use `dict.get` to look up `CONF_TYPE`.

Reply:
```suggestion
    dev_type = config.get(CONF_TYPE)
```

Reply:
If dev_type is None I can't generate the correct remote class using blk.gendevice(). We need a default value here.